src/main/java/ca/bc/gov/backendstartapi/config/CorsConfig.java

```diff
@@ -23,7 +22,7 @@ public class CorsConfig implements WebMvcConfigurer {
    */
   @Override
   public void addCorsMappings(CorsRegistry registry) {
-    if (!ObjectUtil.isEmptyOrNull(allowedOrigins)) {
+    if (allowedOrigins.length != 0) {
```

Comment:
I would rather keep the old check as it will verify possible NullPointerException as well

Reply:
But can an NPE happen here?

- There's a default value for `server.allowed.cors.origins`, so `allowedOrigins` won't ever be empty;
- If we removed the default value for `server.allowed.cors.origins` and provide nothing, we get another error before getting here;
- If we provide an empty string to `server.allowed.cors.origins`, `allowedOrigins` will be empty.